modules/scheduler/executor/plugins/k8s/k8s.go

```diff
@@ -79,11 +79,14 @@ const (
 	MIN_CPU_SIZE = 0.1
 
 	// ProjectNamespace Env
-	LabelServiceGroupID                = "servicegroup-id"
-	KeyServiceGroupID                  = "SERVICE_GROUP_ID"
-	KeyOriginServiceName               = "ORIGIN_SERVICE_NAME"
-	ProjectNamespaceServiceNameNameKey = "PROJECT_NAMESPACE_SERVICE_NAME"
-	ProjectNamespace                   = "PROJECT_NAMESPACE"
+	LabelServiceGroupID = "servicegroup-id"
+)
+
+type RuntimeServiceOperator string
+
+const (
+	RuntimeServiceKeep   RuntimeServiceOperator = "Keep"
+	RuntimeServiceDelete RuntimeServiceOperator = ""
```

Comment:
ambiguous

Reply:
update RuntimeServiceKeep to RuntimeServiceRetain
update RuntimeServceDelete value from "" to "Delete"